test/tz_test/validate.cpp

```diff
@@ -90,9 +90,9 @@ tzmain()
     auto& db = get_tzdb();
     std::vector<std::string> names;
     names.reserve(db.zones.size() + db.links.size());
-    for (auto& zone : db.zones)
+    for (const auto& zone : db.zones)
```

Comment:
`auto&&` a universal reference and will be `const T&` if the container is `const` and non-const otherwise. It's more concise and makes the type less explicit.